Library/Formula/mg.rb

```diff
@@ -1,20 +1,12 @@
 class Mg < Formula
   desc "Small Emacs-like editor"
-  homepage "http://homepage.boetes.org/software/mg/"
-  url "http://homepage.boetes.org/software/mg/mg-20131118.tar.gz"
-  sha256 "b99fe10cb8473e035ff43bf3fbf94a24035e4ebb89484d48e5b33075d22d79f3"
-
-  depends_on "clens"
+  homepage "http://devio.us/~bcallah/mg/"
+  url "http://devio.us/~bcallah/mg/mg-20151130.tar.gz"
+  sha256 "de589aa5b2e5067443a7c3ecf9cea4747056b2ebd6c245e78e029f1a89b62ee1"
 
   conflicts_with "mg3a", :because => "both install `mg` binaries"
 
   def install
-    # makefile hardcodes include path to clens; since it's a
-    # nonstandard path, Homebrew's standard include paths won't
-    # fix this for people with nonstandard prefixes.
-    # Note mg also has a Makefile; but MacOS make uses GNUmakefile
-    inreplace "GNUmakefile", "$(includedir)/clens", "#{Formula["clens"].opt_include}/clens"
-
     system "make"
     bin.install "mg"
```

Comment:
That should work already if we pass `MANDIR` as well. I’ll look into it (probably later this weekend).